src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -458,6 +458,11 @@ private void OnUpdateGraphCompleted(AsyncTask task)
         /// </summary>
         public void Run()
         {
+            if (!RunSettings.RunEnabled)
+            {
+                return;
+            }
+
```

Comment:
Wow this is new, what scenario is this meant to solve? It'll be nice to add a comment to explain why this is important here so future readers.